### ISSUE TITLE
Pagerduty API v1 EOL in February 2018

### ISSUE THREAD

**Issue by claudeatsafe** [librato/librato-services#156]
The Librato pagerduty service relies on the API v1, PagerDuty won't support v1 anymore after February 6, 2018. From their FAQ:
> WHEN DOES THE SUPPORT FOR THE PREVIOUS API VERSION END?
API v1 will be supported until February 6, 2018. If you find yourself in a unique situation that requires an extension, please contact our support team at support@pagerduty.com.
As of July 6, 2016, no new features will be released for v1.

Is Librato aware of this?

**Comment by mheffner** [comment]
@claudeatsafe Yes, we are aware of this and tracking it. We plan to provide updates shortly. Thanks for the notification!
